pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -45,6 +45,13 @@ self: super: {
   # Needs older QuickCheck version
   attoparsec-varword = dontCheck super.attoparsec-varword;
 
+  # http://bugs.darcs.net/issue2642
```

Comment:
Thanks for submitting this upstream.  It looks like the author replied.

Maybe you should link the author to the patch from @nh2?  It might be useful to the author as a starting point to start updating darcs in a backward-compatible way.